js/app.js

```diff
@@ -30,6 +30,18 @@ phpEngine.configure({
 // Expose jQuery to PHPland
 phpEngine.expose($, 'jQuery');
 
+// Expose Window to PHPland
+var this_window = window ;
```

Comment:
Could we just inline these extra variables like `this_window` to keep the code shorter/simpler please? eg. `phpEngine.expose(window, 'window');`